test/test_blockedtuple.jl

```diff
@@ -54,12 +55,36 @@ using TensorAlgebra: BlockedTuple, blockeachindex, tuplemortar
     BlockedTuple{3,blocklengths(bt)}(Tuple(bt) .+ 1)
   @test (@constinferred bt .+ tuplemortar(((1,), (1, 1), (1, 1)))) ==
     BlockedTuple{3,blocklengths(bt)}(Tuple(bt) .+ 1)
-  @test_throws DimensionMismatch bt .+ tuplemortar(((1, 1), (1, 1), (1,)))
+  @test bt .+ tuplemortar(((1, 1), (1, 1), (1,))) isa NTuple{5,Int}
+  @test bt .+ tuplemortar(((1, 1), (1, 1), (1,))) == (2, 5, 3, 6, 4)
```

Comment:
I think ideally this would combine the blocking of the inputs, for example:
```julia
julia> using BlockArrays

julia> mortar([[1, 2], [3, 4]]) + mortar([[1], [2, 3, 4]])
3-blocked 4-element BlockVector{Int64}:
 2
 ─
 4
 ─
 6
 8
```
but I'm fine with the current behavior of removing the blocking for the sake of keeping this PR simpler, we can always add support for combining the blocking if needed.

Reply:
At the very least we should add an issue about this as a reminder.